### ISSUE TITLE
Remove Armor of Faith from the Priest

### ISSUE THREAD

**Issue by Marcloure** [mlenser/kryx-rpg-issues#807]
Armor of Faith lets the Priest get to 20 AC while being a full caster (getting better offensive and defensive spells than gishes). This is the same AC a Paladin has while trading a large amount of DPS to wield a shield, and quite more than other casters.

As a substitute feature, they could instead gain advantage on any skill check related to their deity, since most players would already ask "I'm a Priest of Bahamut, shouldn't I gain advantage on this check?". Advantage can't reveal any obscure knowledge, so their are still limited on that, and it would be similar to how the Suffused gains advantage to interact with their origin creature type.

**Comment by mlenser** [comment]
The priest, like the Avenger and Sacred Fist, is meant to be protected by their deity instead of armor. Armor of Faith fulfills that role for them.

Armor of Faith is does provides 11 + half prof + Dex + Spellcasting (max 3). That's 11+3+5+3 = 22 Defense at 17th level (assuming 5 dex, which isn't a great assumption).

At 17th level, a Paladin has 18 + half prof = 21 Defense. With a Shield that'd be 23 Defense.

Armor is outscaled by these benefits at max level _if_ the secondary/tertiary ability is raised. The same is true for RAW Monk's AC and the other RAW AC providing formulas such as Barbarian's Unarmored Defense which would also scale to be higher than armor if the tertiary ability is invested in.

**Comment by Marcloure** [comment]
> assuming 5 dex, which isn't a great assumption

Why not? Isn't the assumption 5 Dex, 5 Wisdom, and 2 Constitution at 11th level? They could have 5 Str instead of 5 Dex, but unarmored Priests shouldn't have encumbrance problems, so 5 Dex seems better.

Also, your conclusion is that the Paladin, a gish focused on defense, ends up with lower defense then a full caster (which can also cast much better shilding spells).

**Comment by mlenser** [comment]
> The same is true for RAW Monk's AC and the other RAW AC providing formulas such as Barbarian's Unarmored Defense which would also scale to be higher than armor if the tertiary ability is invested in.

**Comment by Marcloure** [comment]
The Monk and the Barbarian aren't full casters though. They can't grant themselves a large bonus to Defense or saves through spells. The Priest has 20 Defense and can gain quite more with shielding spells when needed. Yes, the Monk and the Avenger can cast spells, but using 1 mana to gain extra Defense is much more costy to them than to the Priest (and they can't quite reach the same bonus).

I am done now, I don't have more to say if you don't agree.

**Comment by mlenser** [comment]
> Priest being a caster makes it no different than an Avenger, Sacred Fist, or a Monk.

All have to expend resources to use defensive spells. The priest only has cantrips and mana while the Avenger, Sacred Fist, or Monk might have spells and/or maneuvers, but also has a higher base DPR.

The Divine power source also does not have as many defensive spells. Shield of Faith is the main one and it's better used on a martial than the Priest.

**Comment by mlenser** [comment]
Complete picture:

Casters primary ability will be their spellcasting ability. From there they gain an equal-ish amount of benefits from Dexterity and Consitution. Dexterity provides Defense, half Reflex, and some skills while Constitution provides Health, Fortitude, and miscellaneous benefits like long marching. They're fairly equivalent. So a caster will either have ~5 Dex or ~5 Con at 11th level while their tertiary ability is ~2, changing to ~3 at 15th and 4 at 19th level. Casters could divide this up a bit more equally as well if desired (good choice, imo).

Mage Armor like spells: cost 1 mana and provide ~16 Defense at level 1, up to ~20 Defense at 5th tier if Dex is 4. If Dex is not secondary then they get ~18-19 Defense.
Scale: 18 Defense at level ~3-4 scaling to 20 Defense at the 5th tier
Plate: 19 Defense at level ~5 scaling to 21 Defense at 5th tier
Monk/Sacred Fist: Dexterity is their primary ability and they'll likely have Con or their spellcasting as their secondary. This will provide ~14+spellcasting = ~15-16 Defense at low levels and will scale up to ~22 Defense, but they are quite behind armor until at least 3rd tier when their Dex is 5.
Avenger/Priest: Both will choose between Dex or Con for their secondary or possibly split it between the two. Likely they will scale slower than the Monk/Sacred Fist and they'd give up more to achieve it.

TL;DR: The current Armor of Faith requires a fair investment of abilities, often at the cost of less Con. It also scales much slower than armor. That makes it totally ok as is, I believe.

**Comment by itamarcu** [comment]
I just want to add - the goal of the "unarmored defense" features is to allow the fantasy of a priest, monk, barbarian, etc who does not wear any armor and is still competitive in a fight.  Outclassing normal armor may be a concern (or not), but either way the solution shouldn't be to remove the entire feature, it should be to nerf/rework the feature in a way that fixes the issue while still making it viable to wear robes, a gi, or nothing at all.

**Comment by Marcloure** [comment]
So, I just had an idea while I was lunching. Why not create a Divine Armor spell for Holy/Unholy, so that Priests and Divine Suffused can use? Currently, Suffused from the Divinity power source don't have access to armor spells.

**Comment by mlenser** [comment]
The Suffused Sorcerer/Warlock has access to Leather armor to provide them with Defense. This is a differentiation between the Acolyte and the Suffused and I wouldn't want to make them the same.
Also, it wouldn't work for pure martial Avengers or Sacred Fists and it would be a huge tax on gish or pure spell versions.

**Comment by Marcloure** [comment]
You could leave Armor of Faith on Avenger and Sacred First, they are fine imo

**Comment by mlenser** [comment]
A deity would not protect their Priests less than their Avengers or Sacred Fists.

I've laid out the math above - for a priest to have high Defense thye must sacrifice Con (Health and Fortitude). It's an option, but it shouldn't be assumed and it only starts competing with armor quite late. It is totally fine as far as I see it.

**Comment by Marcloure** [comment]
> A deity would not protect their Priests less than their Avengers or Sacred Fists.

Why they protect the Paladin less then?

**Comment by mlenser** [comment]
> Why they protect the Paladin less then?

They don't, they gave them the ability to wear plate. 😄
